Desktop version completed

kkolade opened the pull request:
Added media queries for 768px breakpoint and went ahead to style the desktop version of the project accordingly

jerryowusu commented on index.html:
```diff
@@ -207,94 +207,93 @@ <h2 class="work__card-title">Multi-Post<br />Stories</h2>
-            alt="angellist logo"
-            class="hero__social-link-image social-links"
-          />
+        <div class="about__bio">
+          <h1 class="about__headline headline-text">About <br />Myself</h1>
+          <p class="about__text-content description-text">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="" class="about__cta-connect cta-primary">Let's Connect</a>
+          <div class="about__social-link">
+            <img
+              src="asset/images/twitter.png"
+              alt="twitter logo"
+              class="hero__social-link-image social-links"
+            />
+            <img
+              src="asset/images/Linkedin icon.png"
+              alt="linked-n logo"
+              class="hero__social-link-image social-links"
+            />
+            <img
+              src="asset/images/medium.png"
+              alt="medium logo"
+              class="hero__social-link-image social-links"
+            />
+            <img
+              src="asset/images/github.png"
+              alt="github logo"
+              class="hero__social-link-image social-links"
+            />
+            <img
+              src="asset/images/angellist.png"
+              alt="angellist logo"
+              class="hero__social-link-image social-links"
```
Certain aspects of the mobile version is not responsive at various screen widths below 768px as depicted in the images below. Kindly ensure that this and other parts of your mobile version still remains responsive for all screen sizes above 375px in order to conform with the figma design. :pray: Kindly check the dimensions like the width, height, positioning, alignment etc.

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-03 08-50-13](https://user-images.githubusercontent.com/65233860/199682616-d11231ae-42aa-40a7-a88e-9bcb55e11012.png)  |  ![Screenshot from 2022-11-03 08-49-15](https://user-images.githubusercontent.com/65233860/199682650-e682e74e-509e-4064-aacc-c22810518817.png)


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-03 08-51-08](https://user-images.githubusercontent.com/65233860/199682901-ea2c052b-cf85-4c16-b76c-6b41ac899b16.png)  |  ![Screenshot from 2022-11-03 09-08-38](https://user-images.githubusercontent.com/65233860/199682922-b06bda60-02cd-4354-87a5-eda1e776c904.png)



 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-03 08-50-32](https://user-images.githubusercontent.com/65233860/199683021-bb3b8ec4-5dfb-4775-92f5-55ed0fc30942.png)  |  ![Screenshot from 2022-11-03 08-49-37](https://user-images.githubusercontent.com/65233860/199683054-1261042e-4df1-4a15-b893-42d350f32b48.png)

jerryowusu commented on css/style.css:
```diff
@@ -356,15 +358,105 @@ section.work {
-/* @media (min-width: 480px) {
-} */
-
 @media (min-width: 680px) {
+  .container {
+    width: 80%;
+  }
```
Kindly note that the desktop version should be 768px or wider instead of 680px in order to conform with the figma design. :+1:

jerryowusu reviewed (changes requested):
Hi @kkolade :wave: 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

![no-we-were-so-close-pauline-turner-brooks](https://user-images.githubusercontent.com/65233860/199685311-1042dbf3-83ce-40c5-8b3d-95cf297c40e4.gif)


## Highlights
All linters are passing :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 
Neat and organised code :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Nice description of yourself in the readme documentation :100: However, the description section of the readme is to explain/describe the project you are working on, and not you, the developer. Kindly check and update accordingly. :+1: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kkolade commented:
- Fixed responsiveness by setting appropriate break-point
- Used appropriate images for all sizes for both header and contact section
- Deleted all unused/commented CSS rules

Owusu-Desmond reviewed (approved):
Hi @kkolade  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
